### PR TITLE
Merge latest from QA to Master

### PR DESCRIPTION
**Pull request by LucasCioffi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
 ## Changelog
+### v0.3.12
```
Comment (ktp242) [inline CHANGELOG.md]: Can we change the version to v0.3.11?

File: README.md [hunk 2]
```diff
@@ -3,17 +3,20 @@
 This repository is the global search application for nypl.org.
 
 ### URL
-https://www.nypl.org/searchbeta
+https://www.nypl.org/search
 
 ### Version
-> v0.3.6
+> v0.3.12
```
Comment (ktp242) [inline README.md]: So is the version here to be aligned with the first release.

File: package.json [hunk 3]
```diff
@@ -1,14 +1,16 @@
 {
   "name": "dgx-global-search",
-  "version": "0.3.6",
+  "version": "0.3.12",
```
Comment (ktp242) [inline package.json]: Also the version here.

Comment (LucasCioffi) [reply package.json]: @ktp242 thanks.  The latest commit now says "1.0.0" in package.json, CHANGELOG, and README: https://github.com/NYPL/dgx-global-search/pull/110/commits/70a3b14bb3f2ec6d2059ecf320025ad7a9069198

**Pull request merged**